### ISSUE TITLE
Port MathCAT to ORCA

### ISSUE THREAD

**Issue by NSoiffer** [NSoiffer/MathCAT#128]
ORCA's math support is old and not great. ORCA is written in Python, but doesn't have an add-on architecture. MathCAT would significantly enhance math accessibility in ORCA.

If someone is part of ORCA or has contacts in ORCA, incorporating MathCAT into ORCA would be really helpful. I suspect it isn't that large a project, but that is based on just a cursory look at ORCA's code. It would likely take advantage of the [MathCATForPython ](https://github.com/NSoiffer/MathCATForPython) interface.

**Comment by NV-Codes** [comment]
This would be very helpful, as Linux is often used for running simulations, in courses that teach computational/numerical techniques, and for high-performance computing, at times including the use of GUI tools for handling data, etc., in which the user uses a desktop environment (like GNOME) instead of the shell alone.

The [Orca GitHub page](https://github.com/GNOME/orca) is a read-only repository, but Joanmarie Diggs (@joanmarie) seems to be the main contributor.

It seems that [preliminary support for MathML was added to Orca in 2015](https://lists.gnome.org/archives/orca-list/2015-July/msg00013.html), but I could not find any recorded changes in Orca's handling of MathML since then. One must manually navigate with the Up and Down arrow keys to a particular line that contains a MathML expression to hear the expression verbalized; interactive navigation is not currently possible.

**Comment by NSoiffer** [comment]
I had a couple of meetings with Joanmarie and two people interested in doing the ports. She suggested a way they could move forward. They went silent after the last meeting a month or two ago. I should ping them to see what's up.
